lib/fetchGlobalID.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Treasure Global ID
+ */
+
+// Modules
+var jsonp = require('jsonp')
+var _ = require('./utils/lodash')
+var invariant = require('invariant')
+
+function cacheSuccess(result, context) {
+  if (context.setCookie) {
+    context.setCookie(context.client.globalIdCookie, result['global_id'])
+  }
+  return result['global_id']
+}
+
+module.exports = function fetchGlobalID (success, error, forceFetch) {
```

Comment:
This function looks pretty trivial, so maybe it's unnecessary, but I don't see where we're testing any of the logic (eg caching, error/success callbacks) of this function.